### ISSUE TITLE
Uncharged energy crystal disappearing on teleport from sewers

### ISSUE THREAD

**Issue by fatal-leo** [scanlime/robot-odyssey-rewired#5]
I collected the black crystal hidden in the sewers.
The (black) uncharged energy crystal was stored partially on the wall of the robot, to make it visible.
After the teleport I immediately entered the robot and the crystal was gone.

I was able to reproduce this unexpeced behaviour with the robot holding the crystal instead, and with the crystal being stored in different places inside the robot.

I did not yet try discharging the charged crystal while inside the sewers.

**Comment by scanlime** [comment]
Yep, I see the same thing here. I suspect it's done on purpose, but it would be interesting to confirm this by checking the game's code!

Here's a save from right before that teleport. The black crystal is on the top wall inside Checkers, but it's gone after the teleport.

https://www.robotodyssey.online/#ESi1L/0EMGUSAMaZUT4gbxsERIhDmYkp7AKSuiB4wA3BVYGCOTD20oDxSBKE1GNsN61lNOlHrau0G8uCLkYz0dSRStsjODF9ekLf91kARAA5ABBDWghwXBlK5uqQRch8dHTcgMVeNBM4QHywrDlJSBxwOCM0FjaYBdaFChQYbGjIFFwPPgZUECTgbCYmSdKcUsppklKaEUIIqRYhF9yCuv2oQJGiACWgWvi2/46/ayKirhUk9q+EQl2nFI7bti3/8R8E+nxej/T/1xAIS8A09rAhMN49gIkuGTRNAwHRRDoeHQ+RzsiIThnTexywDTHxtYqqf3818B5O3v+v+hnvqtn8c3KAElb23NAjt70Qc4mZQtVhfB+Gp9hWKJYBvL4n3uZdXIZoPCrHF9v2xn/9X///9f9fV1VdX/VtXH+6/vD3//9/77fHs2cRJZdMIHy51L8KeqjRXKmmSApAHGkMcZ6SiQgBQQwDksAEJIszDIIzQkIEhggMERAjREIiPCJEgoKOwwHoOaO9yrJ+BvCb6BNMe+KPoFdRgwq5YCGtQmDITZC+mU03Hnx9Tg23yUhyDMJHWH6hE1rVlhHroEHssWqoREV4+35W/EhXqbkVfGqTCHLmbjWF3mHBbOtG9qVGgoKlkPi9C6SEwYd4H4SxnriwqegUDXaqCUQ7Oes44g3g983Ov+QJ/8tb0AwfVpjCDXBwYDnbl3mB9pD+3PAtL8JmAoCmnBx3FEOoHYpjmHUQy6GK/3zgRhD9R0MyNOvjcuFr2TYYNdfCTNMGGLfG+ylXJgYKBccM8O0=

**Comment by fatal-leo** [comment]
Oh, I checked again... This time I collected all the items, discharged checkers [state attached] and used the robot to discharge the charged crystal. That's when I realised that the usual discharged crystals are white. I was going into this with the assumption that I could recharge the crystal in the next level.

If there really are no more Ampire bots in the game, there is indeed no point to carrying a black crystal through it. Thank you for your time :)

https://www.robotodyssey.online/#ESi1L/0EMN0YACZfbEbwsAoFfDi0aZIglxTHaHWIiQ8z8DCPXgHsNfATSprFSnS1YMZCCtzD4KzBZRImB7InrFMemq4VcvMWxYGJN2/O/uDfet8DcwBmAEoADTSQSGWNNqqVYFgRETF7tVbEwF45BxRGSM7zdLlcPB5vtAHnQUXnk6kAs7loH0hMUjjQWDLVyAsAKWISCikgyQSkTMYSRXEcy9BJi+7ODRwdm5sgsGZ3bcioMWOCBYfitmsAwa4OpEBUBD5KA0YBRGLFETARTRnjQsiM9Qj3pZFToKNUsdlOl1wsF3PJ7Xa0lCrVipOtjKLoExKw4MT0nEzT3DRPp82c97xnzzSlL/9/Z3F8towA8uJ74EKCuVqRSLOZBVQGVytEhQ+mB/NSogVEzFmDwILEiNPxrR0qlS1o0PXpG1Pt78zhh+fTSZOOonx9/N0Qp8y7217Hwbm5+bexqakZ9D/+B4NAID38Q/oRrlKY/sWJlmfQ0/2rh/K1/wkBVCdHuO/4/////1NKn9I1VfJKf/G9znKL3+8niv/5Q3/U+BTVEaL+fTotxT/1TkvazxpGScq31WOlL3qUo2oMpwGAsqjxVEtsBgsKSCKgAEGiKhEoAAhVYgEa4kCABiAKMEoBAREAAARAiIRACAQAACAwRqm7wmRio70qff0MsIV8hVfsbC25vYIOlRw8ICoJ0kcpGw1LeDFgJkv76IDPZNSNwAcq29wDw4VLFJZfiLoyXEPwBqG+cu6CGjLhqDvG5b2EWIkL8tu+RID8I1I81PGNUq8ah3PvrVt5A0EN7kmkW1FNRwbbVVQF6ZepaGFyQJQSKR0xdwJYc9UgKkifK/afkbDOVavn260zsgHn9pJPmdkEIO3+PiWMd4CYl3MkEOEu8gLAE5XuA8iGN1OPCIcesfMinDYFCr+t/gg1UDktvAuMlKJCxckHrYvQWWUTei012ztKBI7KWHJcceWz00DVA2BamZBHFENhBkAg2ofPIU3mZMTogggclbGkqQv3ILsg+KORedjdRuzgrgChTtMSVpB7ulC0zMvrc0f9OJDjxv0rk5mKAvPVz+I=
